src/temp_utils.jl

```diff
@@ -11,21 +11,7 @@ E.g. filename("/some/path/to/file_name.ext") -> "file_name"
 filename(path) = splitext(basename(path))[1]
 
 
-# this should be in PropDicts?
-"""
-    propdict(json_file)
-
-AbstractString -> PropDict 
-
-Construct a PropDict based on given <json_file>.
-Alias for `readprops` from PropDicts.jl
-    
-"""
-function propdict(json_file::AbstractString)
-    readprops(json_file)
-end
-
-    
+Base.@deprecate propdict(json_file::AbstractString) readprops(json_file)
```

Comment:
If users of LegendGeSim don't typically use this, let's remove it completely.

Reply:
I force-pushed to remove `propdict` completely in this PR.